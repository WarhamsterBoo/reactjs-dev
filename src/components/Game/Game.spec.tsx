import React from "react";
import renderer from "react-test-renderer";
import { Game } from ".";

describe("Game", () => {
  it("should render ", () => {
    const sut = <Game />;

    expect(renderer.create(sut).toJSON()).toMatchSnapshot();
  });
});
