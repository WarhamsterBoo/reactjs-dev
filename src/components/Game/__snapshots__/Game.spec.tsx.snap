// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game should render  1`] = `
.emotion-118 {
  min-width: 600px;
  min-height: 400px;
  background-color: #454a43;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-content: space-between;
  -ms-flex-line-pack: space-between;
  align-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-9 {
  padding: 10px 10px 10px 10px;
  border: solid 1px black;
  background-color: #011627;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-8 {
  border-color: #addb67;
}

.emotion-0 {
  color: #addb67;
  font-family: Inconsolata,monospace;
}

.emotion-2 {
  margin-right: 10px;
  margin-left: 10px;
  color: #addb67;
  font-family: Inconsolata,monospace;
}

.emotion-1 {
  font-family: Inconsolata,monospace;
  color: #addb67;
  border-radius: 2px;
  border: 1px solid;
  background: transparent;
}

.emotion-7 {
  margin-left: 7px;
  margin-right: 7px;
  border-radius: 4px;
  border: none;
  background-color: #addb67;
  color: #011627;
  font-family: Inconsolata,monospace;
}

.emotion-110 {
  display: inline-block;
  padding: 1px;
  background: #2d302b;
}

.emotion-10 {
  width: 20px;
  height: 20px;
  border: none;
  margin: 1px;
  display: inline-block;
  line-height: 25px;
  text-align: center;
  vertical-align: bottom;
  background: #454a43;
}

.emotion-117 {
  padding: 10px 10px 10px 10px;
  border: solid 1px black;
  background-color: #011627;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div
  className="emotion-118"
>
  <form
    className="emotion-9"
    onSubmit={[Function]}
  >
    <fieldset
      className="emotion-8"
    >
      <legend
        className="emotion-0"
      >
        Game settings
      </legend>
      <label
        className="emotion-2"
      >
        X dimension:
        <input
          className="emotion-1"
          max="100"
          min="1"
          name="xDimension"
          onChange={[Function]}
          type="number"
          value={10}
        />
      </label>
      <label
        className="emotion-2"
      >
        Y dimension:
        <input
          className="emotion-1"
          max="100"
          min="1"
          name="yDimension"
          onChange={[Function]}
          type="number"
          value={10}
        />
      </label>
      <label
        className="emotion-2"
      >
        Filling Percentage:
        <input
          className="emotion-1"
          max="100"
          min="0"
          name="fillingPercentage"
          onChange={[Function]}
          type="number"
          value={0}
        />
      </label>
      <button
        className="emotion-7"
      >
        Apply
      </button>
    </fieldset>
  </form>
  <div
    className="emotion-110"
  >
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <br />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <br />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <br />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <br />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <br />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <br />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <br />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <br />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <br />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <button
      className="emotion-10"
      onClick={[Function]}
    />
    <br />
  </div>
  <div
    className="emotion-117"
  >
    <button
      className="emotion-7"
      id="stop"
      onClick={[Function]}
    >
      Stop
    </button>
    <button
      className="emotion-7"
      id="run"
      onClick={[Function]}
    >
      Run
    </button>
    <button
      className="emotion-7"
      id="pause"
      onClick={[Function]}
    >
      Pause
    </button>
    <button
      className="emotion-7"
      id="faster"
      onClick={[Function]}
    >
      Faster
    </button>
    <button
      className="emotion-7"
      id="normal"
      onClick={[Function]}
    >
      Normal
    </button>
    <button
      className="emotion-7"
      id="slower"
      onClick={[Function]}
    >
      Slower
    </button>
  </div>
</div>
`;
